Formula/emacs.rb

```diff
@@ -1,11 +1,22 @@
 class Emacs < Formula
   desc "GNU Emacs text editor"
   homepage "https://www.gnu.org/software/emacs/"
-  url "https://ftp.gnu.org/gnu/emacs/emacs-27.1.tar.xz"
-  mirror "https://ftpmirror.gnu.org/emacs/emacs-27.1.tar.xz"
-  sha256 "4a4c128f915fc937d61edfc273c98106711b540c9be3cd5d2e2b9b5b2f172e41"
   license "GPL-3.0-or-later"
 
+  stable do
+    url "https://ftp.gnu.org/gnu/emacs/emacs-27.1.tar.xz"
+    mirror "https://ftpmirror.gnu.org/emacs/emacs-27.1.tar.xz"
+    sha256 "4a4c128f915fc937d61edfc273c98106711b540c9be3cd5d2e2b9b5b2f172e41"
+    patch do
```

Comment:
- Add one blank line above each `patch` block
- Add a comment with a one-line explanation about the role of each patch

Reply:
Done.